mmdet/models/necks/nasfcos_fpn.py

```diff
@@ -0,0 +1,202 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import xavier_init
+
+from ..builder import NECKS
+
+from mmdet.ops import ModulatedDeformConvPack
+from mmcv.cnn import ConvModule
+
+
+class MergingCell(nn.Module):
```

Comment:
Since NASFPN already create some cells, is it possible to reuse them or combine them by refactoring?

Reply:
Cells created in NASFPN is kinda hard-coded, and it is difficult to simply reuse them.

Shall I modify the code in NASFPN so as to make cells reuseable, or just maintain the status quo?

Reply:
Yeah, you can modify the code in NASFPN to make the cells reusable.